packages/node/src/builders/execute/execute.impl.spec.ts

```diff
@@ -38,7 +38,8 @@ describe('NodeExecuteBuilder', () => {
       buildTarget: 'nodeapp:build',
```

Comment:
Please add some tests for when `watch` is `false`.

Reply:
Done!

Reply:
Honestly, that can be hard since requires to make a change

Reply:
It's mocked out here so you don't actually need to make a change to the file.

See: https://github.com/nrwl/nx/blob/c151f7b6244ac086bbaf6753bf8b57198e1a68d7/packages/node/src/builders/execute/execute.impl.spec.ts#L162

Reply:
Cool, but still don't know how to do that. Sorry. I'm newbie with NX.